_mydocs/calendar-pro/features/events.md

```diff
@@ -5,4 +5,30 @@ published:          true
 date:               2022-07-17
 modified:           2022-07-18
 order:              /calendar-pro/features/events-holidays
----
\ No newline at end of file
+---
+
+Calendar Pro supports events and holidays like any other calendar does.
+
+This section describes how these features work.
+
+## Events
+In Calendar Pro, events can be one-time or recurrent.
+
+The idea behind the use of events is that they may be useful to intercept relations or correlations.
+
+For example, if you would like to investigate if the sales amount of a product is influenced by the number of deliveries, you may consider the delivery dates as events:
+
+<img src="images/events.png" width="800" alt="Events in Calendar pro">
+
+Learn more about how events can be managed in the [options section](../options/events/index.md).
```

Comment:
Let's write  'Events section' (general rule: we want it to be clear what the target page is from an eye scan).